tests/parsing/test_full_telemetry_parsing.py

```diff
@@ -1,8 +1,75 @@
 import pytest
+from modules.telemetry.v1.block import BlockHeader
+from modules.telemetry.telemetry_utils import parse_radio_block
 from modules.telemetry.telemetry_utils import is_valid_packet_header
 from modules.telemetry.v1.block import PacketHeader
 from modules.misc.config import load_config
 
+LINE1 = "564133494e490000000c0101010000000200020000000000ac5d00000200030000000000f0c300000200030000000000f0c30000"
+
+
+@pytest.fixture
+def pkt_version() -> int:
+    """
+    returns the packet version as an integer
+    """
+
+    return int("0x" + LINE1[19:21], 16)
```

Comment:
If you're going to convert this to an integer I would just put the integer itself, like `3`. You can keep it as a constant. I see you don't use the packet header anywhere.

I would just split the `LINE1` constant into the parts you're using and put them in fixtures.